launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -44,9 +45,8 @@
       <arg name="input_obstacle_and_raw_pointcloud" value="false"/>
       <arg name="input/obstacle_pointcloud" value="/perception/obstacle_segmentation/pointcloud"/>
       <arg name="output" value="/perception/occupancy_grid_map/map"/>
-      <arg name="method" value="laserscan_based_occupancy_grid_map"/>
-      <arg name="map_origin" value="base_link"/>
-      <arg name="scan_origin" value="base_link"/>
+      <arg name="occupancy_grid_map_method" value="laserscan_based_occupancy_grid_map"/>
+      <arg name="occupancy_grid_map_param_path" value="$(var laserscan_based_occupancy_grid_map_param_path)"/>
```

Comment:
@soblin 
Since there are no `laserscan_based_occupancy_grid_map_param_path` declared in this file, so running default psim will fail with no_parameters.

Do you have any idea?

How about add default parameters to `probabilistic_occupancy_grid_map.launch.xml`?

```xml
  <arg name="occupancy_grid_map_method" default="pointcloud_based_occupancy_grid_map" description="options: pointcloud_based_occupancy_grid_map, laserscan_based_occupancy_grid_map"/>
  <arg name="occupancy_grid_map_param_path"default="($ var occupancy_grid_map_method)"/>
```

Reply:
It worked with following autoware_launch PR.
https://github.com/autowarefoundation/autoware_launch/pull/294#pullrequestreview-1383331633